src/features/search/pages/modals/LocationModal/LocationModal.native.test.tsx

```diff
@@ -582,6 +584,39 @@ describe('<LocationModal/>', () => {
         payload: expectedSearchParams,
       })
     })
+
+    it('should navigate when pressing location button on landing page ', async () => {
```

Comment:
`should navigate when pressing submit button from location filter modal to landing page`